solutions/3761/01.py

```
@@ -1,48 +1,21 @@
 class Solution:
-    def countPaths(self, n: int, edges: List[List[int]]) -> int:
-        # Sieve of Eratosthenes to find primes
-        is_prime = [True] * (n + 1)
-        is_prime[0] = is_prime[1] = False
+    def minMirrorPairDistance(self, nums: List[int]) -> int:
```

Ruff (0.14.8)

Undefined name `List`

(F821)